tests/test_pep518.py

```diff
@@ -0,0 +1,24 @@
+import subprocess
+import sys
+
+import pytest
+
+from .conftest import cd_package
+
+
+@pytest.mark.parametrize(
+    ('package'),
+    [
+        'purelib-and-platlib',
+    ]
+)
+@pytest.mark.parametrize(
+    'build_arg', ['', '--wheel'], ids=['sdist_to_wheel', 'wheel_directly']
+)
+@pytest.mark.xfail(sys.platform.startswith('cygwin'), reason='ninja pip package not available on cygwin', strict=True)
```

Comment:
FYI, this fails on Cygwin in this branch (no ninja package), and passes in #175 - it's great to see this test does exactly what is supposed to do! :)